pkg/preflight/builtin_test.go

```diff
@@ -211,7 +176,39 @@ spec:
 			} else {
 				require.NoError(t, err)
 			}
-			assert.Equal(t, tt.want, string(got))
+
+			input := map[string]interface{}{}
+			err = yaml.Unmarshal(got, &input)
+			require.NoError(t, err)
+
+			for _, q := range tt.want {
+				query, err := gojq.Parse(q.query)
+				if assert.NoError(t, err) {
+					iter := query.Run(input)
+					v, ok := iter.Next()
+					if !ok {
+						assert.NoError(t, errors.New("iterator empty"))
+					} else {
+						if err, ok := v.(error); ok {
+							assert.NoError(t, err)
+						} else {
+							b, err := yaml.Marshal(v)
+							if assert.NoError(t, err) {
+								assert.Equal(t, yamlNormalize(t, []byte(q.value)), yamlNormalize(t, b))
+							}
+						}
+					}
+				}
+			}
 		})
 	}
```

Comment:
I don't really understand why this test is being replaced
I might be misreading but the new test seems a lot different than the previous one, and doesn't seem to fit the rest of the PR to me

Reply:
the existing tests forced me to copy paste the entire yaml each new test case. the file started getting extremely long so i refactored the test so you can test an individual property each case